test/sql/test_sink/R/test_fault_injection_hdfs

```diff
@@ -0,0 +1,285 @@
+-- name: test_sink_parquet_fault_injection_hdfs
```

Comment:
it should be sequential ? set failpoint may have impact on other cases

Reply:
done